src/wizards/tokenSwapDealWizard/stages/termsStage/termsStage.html

```diff
@@ -46,5 +46,108 @@ <h2 class="sidebarHeader heading heading3 title" data-test="stageSectionSidebarT
         </p>
       </div>
     </div>
+
+    <div class="stageSection">
+      <!-- Sidebar -->
+      <div class="stageSectionSidebar">
+        <h2 class="sidebarHeader heading heading3 title" data-test="stageSectionSidebarTitle">
+          DAOplomat Reward
+        </h2>
+        <p class="sidebarText">
+          The reward split will incentivize deal makers to bring more deals to the platform and improve the success rate
+          of a deal negotiation.
+        </p>
+      </div>
+
+      <!-- Content -->
+      <div class="stageSectionContent daoplomatRewardsTotalContainer">
+        <pform-input label="Enable DAOplomat Reward">
+          <ptoggle changed.bind="toggleDaoplomatRewards"
+                   value.to-view="daoplomatRewards"
+          ></ptoggle>
+        </pform-input>
+
+        <div if.bind="daoplomatRewards">
+          <pcard>
+            <pform-input
+              class="daoplomatRewardsTotal"
+              input-reference.bind="percentageAmountInput"
+              label="Total DAOplomat Reward"
+              label-description="Percentage of total deal value that will be split among the DAOplomats."
+            >
+              <div class="daoplomatRewardsTotalContent">
+                <pinput-group>
+                  <pinput-numeric
+                    not-wei
+                    value.bind="daoplomatRewards.percentage & validate"
+                    view-model.ref="percentageAmountInput"
+                  ></pinput-numeric>
+                  <template au-slot="after">%</template>
+                </pinput-group>
+
+                <span if.bind="tokensTotal && daoplomatRewards.percentage">~${tokensTotal * daoplomatRewards.percentage | currency} </span>
+              </div>
+
+            </pform-input>
+
+            <div class="daoplomat"
+                 repeat.for="daoplomat of daoplomatRewards.daoplomats"
+            >
+              <let address-input=""></let>
```

Comment:
these two template variables are used to dynamically assign the view-mode reference in the loop. They are created here so they are scoped to the repeat.for loop

Reply:
i meant to say to add the comment in the code itself